lld/test/ELF/emulation-amdgpu.s

```diff
@@ -1,10 +1,16 @@
 # REQUIRES: amdgpu
 
-# RUN: llvm-mc -filetype=obj -triple=amdgcn-amd-amdhsa %s -o %t.o
```

Comment:
we need to keep a test without --amdhsa-code-object-version= option

Reply:
I am replacing the default testing with explicit testing of cov4 and cov5 to reduce the churn in the patch to make cov5 default.